internal/contentcache/contentcache.go

```diff
@@ -61,7 +89,7 @@ func (c *ContentCache) RecoverCache() error {
 // Helper function that matches the format of a gcsfuse file
 func matchPattern(fileName string) bool {
 	// TODO ezl: replace with constant defined in gcsx.TempFile
-	match, err := regexp.MatchString(fmt.Sprintf("gcsfuse[0-9]+[.]json"), fileName)
+	match, err := regexp.MatchString(fmt.Sprintf("gcsfusecache[0-9]+[.]json"), fileName)
```

Comment:
I see the file name formats used by both contentcache.go and temp_file.go. They should be controlled in one place.

Reply:
I will temporarily move this to gcsx, but in reality we really should be using the new package.

Reply:
Next PR will be a refactor.